compiler/rustc_codegen_llvm/src/coverageinfo/mapgen.rs

```diff
@@ -331,12 +331,8 @@ fn add_unused_functions<'ll, 'tcx>(cx: &CodegenCx<'ll, 'tcx>) {
     // Add a new `FunctionCoverage` to the `function_coverage_map`, with unreachable code regions
     // for each region in it's MIR.
 
-    // Convert the `HashSet` of `codegenned_def_ids` to a sortable vector, and sort them.
-    let mut sorted_codegenned_def_ids: Vec<DefId> = codegenned_def_ids.iter().copied().collect();
-    sorted_codegenned_def_ids.sort_unstable();
```

Comment:
Why was this sorted? Should the comment be corrected?

Reply:
I'm not sure why it was sorted before. We'll eventually need to remove this, right?